jbpm-workitems/jbpm-workitems-rest/src/main/java/org/jbpm/process/workitem/rest/RESTWorkItemHandler.java

```diff
@@ -1343,6 +1366,29 @@ protected HttpClient getHttpClient(Integer readTimeout,
         }
     }
     
+    protected BasicCookieStore addCookie(String cookie,String url) {
```

Comment:
format issue: add a blank space after comma.
```suggestion
    protected BasicCookieStore addCookie(String cookie, String url) {
```